encoding.go

```diff
@@ -135,17 +136,49 @@ func parseLeafNode(serialized []byte, depth byte) (VerkleNode, error) {
 	return ln, nil
 }
 
+func parseSkipList(serialized []byte, depth byte) (VerkleNode, error) {
+	var values [NodeWidth][]byte
+	offset := leafStemOffset + StemSize + 3*banderwagon.UncompressedSize // offset in the serialized payload
+	valueIdx := 0                                                        // Index of the value being deserialized
+	for valueIdx < NodeWidth {
+		rangecount := serialized[offset+1]
+		gapsize := serialized[offset]
```

Comment:
Can we rename `gapsize` and `valueIdx` to `rangeSkip` and `rangeCount`, as to use the same names in the serialization `Skip` and `Count`?

Using `gap` or `range` worlds can be confusing. Using the same names as the serialization makes easier to understand the both ways of the algorithm.